python/cudf/cudf/core/column/categorical.py

```diff
@@ -423,6 +458,15 @@ def remove_categories(
         dtype: category
         Categories (2, int64): [1, 2]
         """
+        if inplace:
+            warnings.warn(
+                "The `inplace` parameter in cudf.Series.cat.reorder_categories"
+                " is deprecated and will be removed in a future version of "
```

Comment:
nit: let's be consistent about putting the space at the end or beginning of the line.

Reply:
Done 👍